### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - David Schildböck, Maxime Lestiboudois and Nathan Parisod

### DISCUSSION THREAD

**Discussion by nyxhawk** [heig-vd-dai-course/heig-vd-dai-course#486] [Show and tell]
# Pirate's Plank &rarr; A custom Hangman game
## Description
*Pirate's Plank* is a playful twist on the classic Hangman game, where a pirate's life is at stake! In this version, the pirate teeters on a plank over shark-infested waters, inching closer to a grisly end with each wrong guess.

The game ends when:
- The user (client) correctly guesses the secret word (chosen by the server), saving the pirate.
- The pirate runs out of lives, plunging into the sea to meet the waiting hungry shark.

## Key Features
**Language** &rarr; The initial release (v1) will be in French. If time permits, we may add English support.
**Lives System** &rarr; Each incorrect guess brings the pirate closer to the shark!

## Setup
**GitHub repo** &rarr; [Pirate's Plank](https://github.com/MaximeLesti/DAI-Projet2)
**Last commit** &rarr; [31fdc5e](https://github.com/MaximeLesti/DAI-Projet2/commit/31fdc5e5a72ec4f0152da332a3aaaf3d3b3047a1)
**Authors** &rarr; [David Schildböck](https://github.com/shadowkudo), [Maxime Lestiboudois](https://github.com/MaximeLesti) and [Nathan Parisod](https://github.com/Nat004499)

Stay tuned for updates, and may the pirate’s fate be in your hands!

**Comment by ludelafo** [comment]
Looks good to us! Remember that your server must manage multiple games at the same time.
